docs/operating/reverse-proxy.md

```diff
@@ -0,0 +1,70 @@
+
+> Note: Using different values for `--web.external-prefix` and `--web.route-prefix` can lead to the web UI not working properly if it is accessed directly (without a reverse proxy).
+
+## Examples
+
+Lets look into some example scenarios. All examples are using Nginx as a reverse proxy here.
+
+### Serving Thanos on a subdomain
+
+Serving a Thanos component on the root of a subdomain is pretty straight-forward. Let's say you want to run Thanos querier behind a Nginx server, accessible on domain `thanos.example.com`. A basic Nginx configuration would look like this:
+
+```
+http {
+    server {
+        listen 80;
+        server_name thanos.example.com;
+
+        location / {
+            proxy_pass    http://localhost:10902/;
+        }
+    }
+}
+
+events {}
+```
+
+### Serving Thanos on a sub-path
+
+Things become a little tricky when you want to serve Thanos on a sub-path. Let's say, you want to run Thanos querier behind an Nginx server, accessible on the URL `http://example.com/thanos`. The Thanos web UI depends on it being accessed on the same URL as Thanos itself is listening. This is because the UI needs to know the URL from where to load static assets and what URL to use in links or redirects. If Thanos is behind a reverse proxy, particularly one where Thanos is not at the root, this doesn't work so well.
+
+To tackle this problem, Thanos provides a flag `--web.external-prefix`.
+
+Let's say we have Thanos querier running on the usual port, we need Nginx running with the following configuration:
+
+```
+http {
+    server {
+        listen 80;
+        server_name example.com;
+
+        location /thanos/ {
+            proxy_pass    http://localhost:10902/thanos/;
+        }
+    }
+}
+
+events {}
+```
+
+With this configuration, you can access Thanos querier on `http://example.com/thanos`. Notice that as we are using `http://localhost:10902/thanos/` as the reverse proxy target, every request path will be prefixed with `/thanos`. To make this work we need t run Thanos querier like
```

Comment:
```suggestion
With this configuration, you can access Thanos Query on `http://example.com/thanos`. Notice that because we are using `http://localhost:10902/thanos/` as the reverse proxy target, every request path will be prefixed with `/thanos`. To make this work we need to run Thanos Query like this:
```